examples/hellohttp/java/BUILD

```diff
@@ -33,4 +33,7 @@ k8s_deploy(
         "hello-http-image": ":server",
     },
     template = "//examples/hellohttp:deployment.json",
+    kubectl_args = [
```

Comment:
So this runs during e2e, but I have not idea how to verify that it actually runs properly.

Reply:
You could do a bazel build for the target and then check the produced file to run kubectl does have the expected argument, e.g., something like:
 ```
 bazel build examples/hellohttp/java:staging.apply
 File="./bazel-bin/examples/hellohttp/java/staging.apply"
  if grep -q "--v=10" "$File"; then
    echo "Success, found argument."
    return 0
  else
    echo "Did not find --v=10 arg!"
    exit 1
  fi
```

Reply:
In mine I just copied the original e2e tests and added my flags. It verifies that the test runs with the flags added but does not verify that the new flags were in place in the command.

Reply:
@omadawn how does it verify that the flags run?

Reply:
It doesn't, it just verifies that the command (including any args) succeeds. So if the flags are applied in the correct location it will work. If you supply an invalid flag or say try --dry-run with $@ before the kubectl command it will fail. 

But it will also succeed if the flags are not applied at all so it's not an excellent test, just better than nothing.

Reply:
@nlopezgi in theory this should be easy, but because of the shell scripts we are using, it is not really that trivial ... I will look at it more.

Reply:
@nlopezgi k, another day and I figured out a good entry point.  Lets see how the testing works ;)

Reply:
Implemented that and testing argument passed in via the cli

Reply:
Added tests